src/configuration.h

```diff
@@ -41,8 +41,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 // Select which board is being used. If the outside build environment has sent a choice, just use that
 #if !defined(T_BEAM_V10) && !defined(HELTEC_LORA32)
-#define T_BEAM_V10  // AKA Rev1 (second board released)
-// #define HELTEC_LORA32
+// #define T_BEAM_V10  // AKA Rev1 (second board released)
```

Comment:
Did you intend to include this change?

Reply:
yeah, good point, I'll be more careful in the future!  I've been pretty sloppy letting this boolean flipflop to configuration.h get checked in too often.